packages/genesys-spark-components/src/components/beta/gux-rich-text-editor/gux-rich-text-editor-beta.tsx

```diff
@@ -2,12 +2,12 @@ import { Component, h, JSX, Element, Prop } from '@stencil/core';
 import { hasSlot } from '@utils/dom/has-slot';
 
 /**
- * @slot typographical-emphasis - Slot for typographical actions
- * @slot text-styling - Slot for text-styling actions
- * @slot lists-indentation - Slot for lists and indentation actions
- * @slot inserting - Slot for inserting actions
- * @slot global-actions - Slot for global actions
- * @slot editor - Slot for the editor
+ * @slot typographical-emphasis - Slot for typographical actions.
+ * @slot text-styling - Slot for text-styling actions.
+ * @slot lists-indentation - Slot for lists and indentation actions.
+ * @slot inserting - Slot for inserting actions.
+ * @slot global-actions - Slot for global action.
```

Comment:
is there a reason why it's global-actions in some areas and global-action in others?

Reply:
In the design it was layed out as if global actions was more than one action but only recently that changed so that global action is just one singular action instead of multiple. I was hesitant to change to as it may be a breaking change but I do not think any teams are using the component yet.